bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeBToTH_Hbb_LH/TprimeBToTH_Hbb_LH_MT600_MH75/TprimeBToTH_Hbb_LH_MT600_MH75_customizecards.dat

```diff
@@ -0,0 +1,3 @@
+set param_card mass 6000006 600
+set param_card ktlh 3 0.100000e+00
+set param_card mass 25 75
```

Comment:
We assume narrow resonances as does the paper we want to probe (https://inspirehep.net/literature/1750464), therefore the default width of 5 GeV for the T should be fine for all mass points. The same holds for the H/a, which uses 4.07e-03 GeV.

Reply:
I agree it will not impact your kinematics. But usually I would add it here so that someone picking up the cards but not probing a fixed width is aware and changes the setting. My experience is that such details get easily lost and the fact that cards were used in the past give people a wrong feeling of doing the correct thing. So if you don't mind I would add two lines
set param_card decay 25 1e-03 
set param_card decay 600006 5 
and then this becomes clear for everyone else. 
One more point, did you check that keeping the width stable has no impact on your cross sections.

Reply:
I have added 
```
set param_card decay 25 4.07e-03
set param_card decay 600006 5
```
to the customizecards in the latest commit. These are the default values of the model. 

Regarding your question, we are using the NWA and hence factorize the T production cross section and its decay. The results of our analysis will also only be interpreted in the NWA, our primary goal is to set upper limits on the cross section x BR. Theoretical cross section calculations for certain benchmark models may be shown for comparison, but these cards will not be used to quote a cross section.